.vscode/settings.json

```diff
@@ -2,4 +2,5 @@
     "files.insertFinalNewline": true,
     "files.trimFinalNewlines": true,
     "files.trimTrailingWhitespace": true,
+    "python.formatting.provider": "yapf",
```

Comment:
Please no, it's by gewgle! 🖤 black 🖤 all the way!
(not that I have any say in your personal projects)

Reply:
already changed~ why black is better than yapf